### ISSUE TITLE
Surface Pro 2017, when the power is plugged in it instantly bugs wifi shutting it down, also after some time the wifi will drop even if the power isnt plugged in

### ISSUE THREAD

**Issue by juaolobo** [jakeday/linux-surface#511]

**Comment by realSpok** [comment]
This looks like a power management issue. What is your network handler ? NetworkManager ?

**Comment by juaolobo** [comment]
Yeah NetworkManager, I don’t see how though I already tried altering the wifi.powersave settings and nothing happened

**Comment by mumudididi** [comment]
i used the command form[ #420]( https://github.com/jakeday/linux-surface/issues/420) and it solved my wifi issue

**Comment by farbod1277** [comment]
I'm having the same issue! It seems like the wifi card is no longer recognized after a while you use it and then you have to reboot! It's super annoying.
